project/Build.scala

```diff
@@ -15,6 +15,9 @@ object KafkaUtilsBuild extends Build {
 
     mergeStrategy in assembly := {
       case "about.html" => MergeStrategy.discard
+      case "jetty-dir.css" => MergeStrategy.discard
```

Comment:
@joaomaiaduarte I ended up taking a slightly different approach to excluding the `pom.properties` files and used the pattern we discussed for `jetty-dir.css` after running into the following during the build:

```
[error] (*:assembly) deduplicate: different file contents found in the following:
[error] /root/.ivy2/cache/com.signalfx.public/signalfx-codahale/jars/signalfx-codahale-0.2.1.jar:jetty-dir.css
[error] /root/.ivy2/cache/org.eclipse.jetty/jetty-util/jars/jetty-util-8.1.13.v20130916.jar:jetty-dir.css
```
Since the plugins intended use is to simply report metrics to Signalfx I'm assuming we can just ignore the signalfx-codehale css file.